.github/actions/spelling/line_forbidden.patterns

```diff
@@ -30,10 +41,22 @@
 \b[Nn]o[nt][- ]existent\b
 
 # s.b. preexisting
-[Pp]re-existing
+[Pp]re[- ]existing
+
+# s.b. preempt
+[Pp]re[- ]empt\b
 
 # s.b. preemptively
-[Pp]re-emptively
+[Pp]re[- ]emptively
+
+# s.b. reentrancy
+[Rr]e[- ]entrancy
+
+# s.b. reentrant
+[Rr]e[- ]entrant
+
+# s.b. workaround(s)
+\bwork[- ]arounds?\b
```

Comment:
Generally this is either mostly wrong or mostly right. It depends on whether a project uses the idea as a noun or a verb. This project appears to be consistently using it as a noun -- where it should be a single word. As such, I'm leaving this enabled. If it turns out that the other use is introduced, you can just disable it w/ `#` as above.

Reply:
Got it!